### DISCUSSION TITLE
git pull in every folder

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [community/community#121345] [New to GitHub] [answered]
how do i do git pull command in every folder in current directory

**Comment by dane** [comment]
If you're running Mac, Linux, BSD, or Windows WSL, and you only have git repos in your current directory, this should work for you:

```
find . -type d -print -exec git -C {} pull \;
```

The `find` command lists all directories. `-exec` calls `git` and replaces `{}` with each directory name. The `-C` tells git which directory to use when pulling.

**Reply by pizzav-xyz** [reply]
what command can i use on windows?

**Comment by ABDE-LKADER** [comment, marked as answer]
For Windows You Can Try This One

`for /d %I in (*) do cd %I && git pull && cd ..`

**Reply by dane** [reply]
@kittenvr does this answer your question?

**Reply by Adrian198802** [reply]
Tak

**Comment by Adrian198802** [comment]
👈

**Reply by pizzav-xyz** [reply]
what are you pointing at

**Comment by QKQS** [comment]
git账户用户名和密码给忘了，百度说在GitHub上面能够重置，应该怎么样进行重置密码？我到现在为止还没有找到办法进行git仓库的代码使用，请求大佬给点意见和建议。

**Reply by ghostinhershell** [reply]
Hi @QKQS, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

**Comment by BaselAbuHamed** [comment]
for /d %I in (*) do cd %I && git pull && cd ..

**Comment by faiyun123** [comment]
****

**Comment by eissa11** [comment]
[د
